docker/frontend/docker-entrypoint.d/99-configure-letsencrypt.sh

```diff
@@ -2,8 +2,30 @@
 
 set -eu
 
-if [ -d "/etc/letsencrypt/live" ]; then
+entrypoint_log() {
+    if [ -z "${NGINX_ENTRYPOINT_QUIET_LOGS:-}" ]; then
+        echo "$@"
+    fi
+}
+
+# display config
+entrypoint_log "$0: Certbot email = ${CERTBOT_EMAIL}";
+entrypoint_log "$0: Certbot domain = ${CERTBOT_DOMAIN}";
+
+# request and install cert
+if [ -d "/etc/letsencrypt/live" ] && [ "${CERTBOT_DOMAIN}" != "_" ]; then
   if ! grep -q 'managed by Certbot' /etc/nginx/conf.d/default.conf; then
```

Comment:
So there is no mechanism for renewing?

I guess for the time being it's ok, I don't think in the foreseeable future any deployment will outlive the 3m renewal time frame. But it's something we'll have to fix in the years to come.

Reply:
This is only at time of starting the container: it will check if it needs to renew it then (usually not needed) and apply correct nginx config.

When the container is running, we can add a cron job that checks periodically for cert renewals:
`certbot renew --nginx`

I will add this as a commit to this PR.